exporter.py

```diff
@@ -22,8 +22,10 @@ def parse_params():
                       dest="collectors")
     parser.add_option("-m", "--config", help="path to config to set default collectors, statkeys and properties for "
                                              "collectors", action="store", dest="config")
+    parser.add_option("-t", "--target", help="define target vcenter", action="store", dest="target")
```

Comment:
The way I got it this should be the target vrops, shouldn't it?

Reply:
Your right. This have to be `vrops` instead of `vcenter`.